hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -965,16 +969,30 @@ public void testDistributedTestDataSource() {
     assertEquals(1000, c);
   }
 
-  private static void prepareParquetDFSFiles(int numRecords) throws IOException {
-    String path = PARQUET_SOURCE_ROOT + "/1.parquet";
+  protected static void prepareParquetDFSFiles(int numRecords, String baseParquetPath) throws IOException {
+    prepareParquetDFSFiles(numRecords, baseParquetPath, "1.parquet");
+  }
+
+  protected static void prepareParquetDFSFiles(int numRecords, String baseParquetPath, String fileName) throws IOException {
+    String path = baseParquetPath + "/" + fileName;
```

Comment:
`Paths.get()` should be better?

Reply:
Paths.get() returns java.nio.file.Paths but we need Hadoop fs path(line 980) here. Hence leaving it as is.